cdap-ui/app/cdap/components/PreviewData/DataView/Table.tsx

```diff
@@ -28,6 +33,19 @@ import classnames from 'classnames';
 
 const I18N_PREFIX = 'features.PreviewData.DataView.Table';
 
+const CustomTableCell = withStyles((theme) => ({
+  head: {
+    backgroundColor: theme.palette.grey['300'],
+    color: theme.palette.common.white,
```

Comment:
Do these colors give enough contrast for readability?

Reply:
Looks like it might be an issue: https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=BBBBBB We want at least 4.5:1 for small text.

Reply:
Good call, and thank you for including the link! I've created a JIRA for this: https://issues.cask.co/browse/CDAP-17304

It might make sense to include checking/improving contrast be part of the component library efforts. I suspect there are a few other components that use the same styling for headers besides in Preview and in the Plugin documentation table.